.github/workflows/ccpp.yml

```diff
@@ -110,7 +116,148 @@ jobs:
           set -x
           exec 2>&1
           abs_top_builddir="$PWD"
+          pkg-config --version
+          pkg-config --help
+          pkg-config --about ||:
+          pkg-config --path libexif ||:
+          pkg-config --path libgphoto2 ||:
+          export PKG_CONFIG_PATH="${abs_top_builddir}/__prefix/lib/pkgconfig:$PKG_CONFIG_PATH"
+          pkg-config --path libexif ||:
+          pkg-config --path libgphoto2 ||:
+          export PKG_CONFIG_PATH="${abs_top_builddir}/__prefix/lib/pkgconfig:"
+          pkg-config --path libexif ||:
+          pkg-config --path libgphoto2 ||:
+          export PKG_CONFIG_PATH="${abs_top_builddir}/__prefix/lib/pkgconfig"
+          pkg-config --path libexif ||:
+          pkg-config --path libgphoto2 ||:
+          export LD_LIBRARY_PATH="${abs_top_builddir}/__prefix/lib"
+          cd examples/ambs-lgp2-frontend
+          autoreconf -vis
+          ./configure --prefix="$PWD/__pref"
+          make
+          ${LDD-false} ambs-lgp2-frontend
+          ./ambs-lgp2-frontend
+          make install
+          ${LDD-false} __pref/bin/ambs-lgp2-frontend
+          __pref/bin/ambs-lgp2-frontend
+
+  msys2-build:
+    needs: ix-build
+
+    runs-on: ${{ matrix.os }}
+
+    name: '${{ matrix.os }} ${{ matrix.sys }}'
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest]
+        # FIXME: mingw32 cannot find WSAStartup() in -lws2_32
+        sys: [mingw64, ucrt64, clang64]
+
+    defaults:
+      run:
+        shell: 'msys2 {0}'
+
+    steps:
+      - name: 'git config core.autocrlf (Windows)'
+        run: git config --global core.autocrlf input
+        shell: bash
+
+      - uses: actions/checkout@v3
+
+      - name: 'Set MINGW_ARCH (Windows)'
+        shell: bash
+        run: echo MINGW_ARCH="${{ matrix.sys }}" >> $GITHUB_ENV
+
+      # See https://github.com/msys2/setup-msys2
+      - name: 'Set up MSYS2 and install build requirements (Windows)'
+        uses: msys2/setup-msys2@v2
+        with:
+          msystem: ${{ matrix.sys }}
+          update: true
+          install: >-
+            git
+            autoconf
+            automake
+            gettext-devel
+            libtool
+            make
+            pkgconf
+            msys2-w32api-headers
+            msys2-w32api-runtime
+          pacboy:
+            gcc:p
```

Comment:
Should be "cc" - this will give you gcc in mingw64/ucrt64 ens, but clang in clang64 env